memo_app.rb

```diff
@@ -3,44 +3,48 @@
 require 'sinatra'
 require 'sinatra/reloader'
 require 'cgi'
-require 'json'
-require 'securerandom'
-
-MEMOS_FILE = 'memos.json'
+require 'pg'
 
 not_found do
   'ページが見つかりません!'
 end
 
-def load_memos
-  JSON.parse(File.read(MEMOS_FILE))
+def connect_db
+  @connect_db ||= PG.connect(dbname: 'memo_app')
+end
+
+configure do
+  result = connect_db.exec("SELECT * FROM information_schema.tables WHERE table_name = 'memos'")
+  if result.values.empty?
+    connect_db.exec('CREATE TABLE memos (
+        id UUID DEFAULT gen_random_uuid() PRIMARY KEY,
+        title VARCHAR(255) NOT NULL,
+        content TEXT,
+        created_at TIMESTAMP WITH TIME ZONE NOT NULL,
+        updated_at TIMESTAMP WITH TIME ZONE NOT NULL
+      )')
+  end
 end
 
-def add_memo(params)
-  memos = load_memos
-  memos[SecureRandom.uuid] = { 'title' => params['title'], 'content' => params['content'], 'created_at' => Time.now, 'updated_at' => Time.now }
-  save_memo(memos)
+def load_memos
+  connect_db.exec('SELECT * FROM memos ORDER BY updated_at DESC')
 end
 
-def update_memo(params)
-  memos = load_memos
-  memos[params['id']].merge!('title' => params['title'], 'content' => params['content'], 'updated_at' => Time.now)
-  save_memo(memos)
+def add_memo(title, content)
```

Comment:
項目が増えると無限に引数が増えていくので、ハッシュで渡してしまった方がいいと思います。

```suggestion
def add_memo(params)
```